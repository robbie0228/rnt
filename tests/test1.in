board
move a down
move A up
board
