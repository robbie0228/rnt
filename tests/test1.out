abc..fgh
...de...
........
........
........
........
...DE...
ABC..FGH
.bc..fgh
a..de...
........
........
........
........
A..DE...
.BC..FGH
